src/components/Shared/HeaderBanner/HeaderBanner.css

```diff
@@ -0,0 +1,49 @@
+  .main-container {
+    height: 100%;
+    position: relative;
+    color: rgba(255, 255, 255, 0.8);
+  }
+  
+  .main-container .main-text {
+    color: rgba(255, 255, 255, 0.9);
+    font-size: 3vh;
+    text-transform: none;
+    padding-top: 2%;
+    padding-bottom: 0%;
+  }
+  
+  .title-text-banner {
+    color: rgba(255, 255, 255, 0.9);
+    font-size: 7vh;
+    text-transform: none;
+  }
+  
+  .main-container .footer {
+    font-size: 2.5vh;
+    text-transform: none;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    position: absolute;
+    bottom: 5vh;
+    left: 0;
+    right: 0;
+    margin-left:auto;
+    margin-right:auto;
+  }
+  
+  .main-container .footer p {
+    margin: 0;
+  }
+  
+  .main-container .footer .icon-btn {
```

Comment:
Delete icon-btn class styling, as well as its hover plz

Reply:
I left the .icon-btn class and removed only the color property instead. Now the .js file manage the color of the icon.